README.md

```diff
@@ -40,11 +40,39 @@ You can test the application with `npm run test` which will run [jest](https://f
 
 You also can have the tests running all the time with `npm run test-watch` which will run `jest --watch`, which sets up jest to automatically figure out which tests to run based on the files you're working with. It's super fast and a great way to immediately tell if you've broken anything.
 
+#### Snapshot testing
+
+Jest allows you to make a really simple tests called "Snapshots" for React components. They're basically like tiny rendered versions of the HTML from the React components. 
+
+A simple snapshot test looks like this:
+
+```js
+import renderer from "react-test-renderer";
+
+test("renders correctly", () => {
+    const tree = renderer
+      .create(
+        <MyReactThing/>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+```
+
+When you run your tests, if a snapshot has been changed it will alert you. If you meant to do this, then you can update automagically it by pressing "u". (It will have a prompt you can follow). 
+
+If you didn't mean to do this, then go fix your things.
+
+You can learn more about snapshot testing here. 
+
```

Comment:
Does this still work well for elements that have some content created dynamically that might be inconsistent?

Reply:
Assuming you give the React component the same inputs/props each time, yes.  React components are stateless and should have no side effects. So if, in your test, you give it:

```js 
const element = <MyReactComponent dynamicStuff={SameEachTest} />
```

You will always get the same snapshot unless you change the `MyReactComponent`.